<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="DESKTOP-EQ55Q8H" failures="1" tests="1" name="Test(failed)" time="0.116" errors="0" timestamp="2022-06-29T14:42:06 IST">
  <testcase classname="TestNg1.disadvantagesOfHardAssert" name="mymethod" time="0.025">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at TestNg1.disadvantagesOfHardAssert.mymethod(disadvantagesOfHardAssert.java:11)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- mymethod -->
  <testcase classname="TestNg1.disadvantagesOfHardAssert" name="my1" time="0.0">
    <ignored/>
  </testcase> <!-- my1 -->
  <testcase classname="TestNg1.disadvantagesOfHardAssert" name="my2" time="0.0">
    <ignored/>
  </testcase> <!-- my2 -->
</testsuite> <!-- Test(failed) -->
